.gitignore

```diff
@@ -16,3 +16,9 @@ TAGS
 *~
 .#*
 .lib
+
+# Intellij-IDEA specific
+.idea
+*.iml
+*.ipr
+*.iws
```

Comment:
You can also add these you your [global git ignore](https://stackoverflow.com/questions/7335420/global-git-ignore) configuration.